rag_api/main.py

```diff
@@ -51,18 +55,19 @@ async def startup_event():
     """Cache vector store at start-up."""
     try:
         initialize_vectorstore(document_location, "**/*.context")
+        global vector_store
```

Comment:
You shouldn't have to declare this as global since you are not changing the variable right?

"Global keyword is used to modify the global variable outside its current scope and meaning. It is used to make changes in the global variable in a local context. The keyword ‘Global’ is also used to create or declare a global variable inside a function."

https://www.simplilearn.com/tutorials/python-tutorial/global-variable-in-python

Reply:
yup, agree